_sass/_post.scss

```diff
@@ -6,7 +6,7 @@
   }
 }
 
-.badge {
+.post-tag {
   display: inline-block;
   padding: .25em .4em;
   font-size: 75%;
```

Comment:
Since these rules are now applied to `<li>` elements, some additional styling will need to occur to remove default margin/padding and display will need to be set to `inline`.

I’ll be happy to style that myself, though.